doc/docker/Dockerfile-solr

```diff
@@ -18,5 +20,8 @@ RUN mkdir -p /opt/solr/server/ez/template \
 # Set our core config as home
 ENV SOLR_HOME /opt/solr/server/ez
 
-# Make sure core is created on startup
-CMD ["solr-create", "-c", "collection1", "-d", "/opt/solr/server/ez/template"]
+# Make sure cores are created on startup
+RUN bin/solr -s ez -p "$SOLR_PORT" \
+ && bin/solr create_core -c collection1 -d server/ez/template -p "$SOLR_PORT" \
```

Comment:
is there anyway we can inject these so all these changes can be applied in general on ezplatform meta repo?